cpp/mcap/include/mcap/errors.hpp

```diff
@@ -23,6 +23,7 @@ enum class StatusCode {
   DecompressionFailed,
   DecompressionSizeMismatch,
   UnrecognizedCompression,
+  UnsupportedCompression,
```

Comment:
would there perhaps be a good reason to keep new status codes monotonically increasing (not modifying any existing status codes)? I'm not sure – just flagging as something to think about

Reply:
Good idea.